style.css

```diff
@@ -0,0 +1,195 @@
+html {
+  box-sizing: border-box;
+}
+
+* {
+  box-sizing: inherit;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+ul li {
+  list-style: none;
+  box-sizing: unset;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+}
+
+/* --------- Header  -------- */
+
+.menu {
+  display: none;
+}
+
+/* --------- navigation menu ------- */
+.nav-menu {
+  display: flex;
+  justify-content: center;
+}
+
+.nav-menu ul {
+  display: flex;
+  width: fit-content;
+  position: fixed;
+  margin: 20px 0;
+}
+
+.nav-menu li {
+  padding: 0 20px;
+}
+
+.nav-menu a {
+  text-decoration: none;
+  color: black;
+}
+
+/* --------- hero section ------- */
+
+.hero {
+  display: flex;
+  background-image: url(/portifolio-website/image/one.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+  min-height: 100vh;
+  justify-content: center;
+  padding: 150px 320px;
+}
+
+.hero .headline {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.place {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  margin: 0;
+}
+
+.primary h2 {
+  font-size: 35px;
+  margin: 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  color: #172b4d;
+}
+
+.primary span {
+  color: #36b37f;
+}
+
+.support {
+  display: flex;
```

Comment:
- [ ] DRY stands for "Don't Repeat Yourself." The main idea with DRY in CSS is to avoid repeating the same bits of code in our style sheet. but you are repeating display flex many times.